frontend/src/static/css/style.scss

```diff
@@ -274,11 +274,15 @@ header{
         }
 
         .untouched{
-            $gray: mui-color('grey', '400');
+            $grey: mui-color('grey', '400');
 
             &.active{
                 .code{
                     display:block;
+                    color:$grey;
+                    [class*='hljs-']{
+                      color:inherit;
+                    }
```

Comment:
dun need to write this, the css on line 302 will take care of it

Reply:
when calling the highlight function in authorship.js, it will add the hljs element which overwrites the parent's implementation. hence this is necessary.

but I can probably leave this code out if prof prefers the syntax highlight to remain on the untouched code.